pisces/stations.py

```diff
@@ -0,0 +1,291 @@
+import numpy as np
+from sqlalchemy import func, or_
+from obspy.core import UTCDateTime, Stream
+import obspy.geodetics as geod
+
+from pisces.io.trace import wfdisc2trace
+from pisces.util import make_wildcard_list, _get_entities
+
+import warnings
+
+def filter_network(query, net=None, netname=None, auth=None, sta=None,  time_=None, **tables):
+    """ipython
+    Parameters
+    ----------
+    session : sqlalchemy.orm.session instance, bound
+    network : 
+    nets : 
+    affiliation : 
+    stas :
+    time_ :
+    endtime : 
+    pref_nets : 
+    with_query :
+    site_name :  
+    Returns
+    -------
+    query
+        sdfs
```

Comment:
What is `sdfs`?

Reply:
lol...just filler characters so I can see that spacing/formatting

Reply:
Docstrings more appropriately filled out